cluster/images/kubemark/BUILD

```diff
@@ -16,6 +16,7 @@ docker_push(
     repository = "$(PROJECT)/kubemark",
     stamp = True,
     tag = "latest",
+    tags = ["manual"],
```

Comment:
what does this do?

Reply:
skips the target from `bazel build //...`, for manual runs we'll `--define PROJECT=foo`

https://docs.bazel.build/versions/master/bazel-user-manual.html#target-patterns
```
Targets with tags=["manual"] will not be included in wildcard target patterns (..., :*, :all, etc). You should specify such test targets with explicit target patterns on the command line if you want Bazel to build/test them.
```